### PR TITLE
New version: SomePythonThings.WingetUIStore version 3.0.1

### LINKED ISSUE #145079: [Update Request]: WinGetUI 3.0.1
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

SomePythonThings.WingetUIStore

### Package Version

3.0.1

### Please describe the changes you would like to see

New version (3.0.1), can be found here: https://github.com/marticliment/WingetUI/releases

Download link: https://github.com/marticliment/WingetUI/releases/download/3.0.1/WingetUI.Installer.exe

### PR DESCRIPTION
**Pull request by drshtmstry**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145085)

### CODE REVIEW

File: manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (vikingnope) [inline manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml:18]:
```suggestion
- Architecture: x64

```

File: manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART /SP- /NoAutoStart
+UpgradeBehavior: install
+Installers:
```
Comment (vikingnope) [inline manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml:17]:
```suggestion
ReleaseDate: 2024-03-16
Installers:

```

File: manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  - The backend has been improved.
+    Packages now load faster and more reliably, providing a more straight-forward experience.
+  - Importing and exporting packages got a big upgrade: Lists of packages can now be opened, edited and exported again.
+    A new Package Bundles section has been added to the interface.
+  New Contributors
+  - @Pandoriaantje made their first contribution in #1687
+  - @yrjarv made their first contribution in #1744
+  - @Malus-risus made their first contribution in #1813
+  - @dave-sc made their first contribution in #1875
+ReleaseNotesUrl: https://github.com/marticliment/WingetUI/releases/tag/3.0.0
```
Comment (vikingnope) [inline manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.locale.en-US.yaml:55]:
```suggestion
ReleaseNotesUrl: https://github.com/marticliment/WingetUI/releases/tag/3.0.1
```

File: manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
```
Comment (vikingnope) [inline manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml:17]:
```suggestion
ReleaseDate: 2024-03-16
Installers:
```

File: manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SomePythonThings.WingetUIStore
+PackageVersion: 3.0.1
+InstallerLocale: en-US
+InstallerType: inno
+Scope: user
+InstallModes:
+- interactive
```
Comment (vikingnope) [inline manifests/s/SomePythonThings/WingetUIStore/3.0.1/SomePythonThings.WingetUIStore.installer.yaml:10]:
```suggestion
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code 1

(Automated response - build 870.)

**Comment by vikingnope** [comment]
@stephengillie seems to be fine now, can be merged

**Comment by stephengillie** [comment]
Sorry for not noticing earlier, but https://github.com/microsoft/winget-pkgs/pull/144886 was submitted by this package's usual uploader.

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #144886;

**Pull request closed without merging**